Sources/App/Controllers/PackageController+BuildsRoute.swift

```diff
@@ -52,20 +52,19 @@ extension PackageController {
             var swiftVersion: SwiftVersion
             var platform: Build.Platform
             var status: Build.Status
+            var docStatus: DocUpload.Status?
 
             static func query(on database: Database, owner: String, repository: String) async throws -> [BuildInfo] {
-                try await Joined4<Build, Version, Package, Repository>
-                    .query(on: database)
-                    .filter(Version.self, \Version.$latest != nil)
-                    .filter(Repository.self, \.$owner, .custom("ilike"), owner)
-                    .filter(Repository.self, \.$name, .custom("ilike"), repository)
+                try await Joined5<Build, Version, Package, Repository, DocUpload>
+                    .query(on: database, owner: owner, repository: repository)
```

Comment:
This means an additional join, and not only on the builds page but also on the package-show page, which reuses this query for the matrix.

I've tested the query with `EXPLAIN ANALYZE` locally and the perf hit is negligible due to the cardinality of max 1 in the join.

I also looked into adding a perf test for that query to our perf test suite but that's not straightforward for Fluent queries. We'd need to find a way to render them into SQL which we can then run through `EXPLAIN ANALYZE`.